README.md

```diff
@@ -68,49 +68,81 @@ trigger OppRollup on Opportunity (after insert, after update,
             new LREngine.RollupSummaryField(
                                             Schema.SObjectType.Account.fields.AnnualRevenue,
                                             Schema.SObjectType.Opportunity.fields.Amount,
-                                            LREngine.RollupOperation.Sum 
-                                         )); 
+                                            LREngine.RollupOperation.Sum
+                                         ));
      ctx.add(
             new LREngine.RollupSummaryField(
                                             Schema.SObjectType.Account.fields.SLAExpirationDate__c,
                                                Schema.SObjectType.Opportunity.fields.CloseDate,
                                                LREngine.RollupOperation.Max
-                                         ));                                       
-	 
-     /* 
-      Calling rollup method returns in memory master objects with aggregated values in them. 
-      Please note these master records are not persisted back, so that client gets a chance 
+                                         ));                                      
+  
+      /*
+        To Prepare MasterId's incase if the realtionship is changed on child object Old Master record and nNew Master Records
```

Comment:
To Prepare MasterId's incase if the realtionship is changed on child object Old Master record and New Master Records